test/test-assert-subroutine-error-termination.F90

```diff
@@ -27,7 +34,8 @@ program test_assert_subroutine_error_termination
 #elif _CRAYFTN
     command = "fpm run --example false-assertion --profile release --compiler crayftn.sh --flag '-DASSERTIONS' > /dev/null 2>&1", &
 #else
-    command = "echo 'example/false_assertion.F90: unsupported compiler' && exit 1", &
+    ! For all other compilers, we assume that the default fpm command works
+    command = "fpm run --example false-assertion --profile release --flag '-DASSERTIONS -ffree-line-length-0' > /dev/null 2>&1", &
```

Comment:
This is an unsafe assumption. This test hard-codes fragile compiler-specific commands, and conflates command failure with test success. Thus the addition of every new compiler needs to be manually verified, and should never be defaulted in this manner.

Please revert this change and add a new block with a command has been verified for correctness with LFortran.